messages/messages.go

```diff
@@ -64,7 +64,7 @@ var (
 	INPUT_OAUTH_TOKEN = "oAuth Token: "
 
 	REGISTRY_AUTH_SUCCESS        = "Successfully authenticated to %s\n"
-	REGISTRY_AUTH_FAIL           = "Failed to authenticate to the registry, this can occur when registry is offline. Until authenticated you will not be able to push new images to your Airflow clusters\n"
+	RegistryAuthFail             = "\nFailed to authenticate to the registry, do you have docker running?\nThis error can occur when docker is not running or the registry is offline.\nUntil authenticated you will not be able to push new images to your Airflow clusters\n"
```

Comment:
let's keep UPPER_CASE so it can be consistent, i know it's contradict to go styling, but we need follow up ticket to fix other places too.

Reply:
Created the follow up ticket for this https://github.com/astronomer/issues/issues/2026.  Should I revert this change or leave as-is and keep this style going forward and make all the other changes within the new issue?

Reply:
it's okey.